src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/contact/ContactRepository.kt

```diff
@@ -26,11 +27,13 @@ data class Contact(
   val name: String,
 
   @NotNull
+  @Column(name = "prison_code")
   val prisonCode: String,
```

Comment:
JPA is mapping `prisonCode` to `prison_code`. When I did the original db schema I'd assumed it would be `prisoncode`. There is a flyway change later down in this PR, and the `@Column` is largely unnecessary but is me being explicit